### PR TITLE
show calls from all available callers

### PR DESCRIPTION
**Pull request by northwestwitch**
I changed the name of the bcftools constant into "Bcftools" otherwise under the gt calls "SAMtools" gets printed twice.

### CODE REVIEW

**Comment by northwestwitch** [comment]
fix #1126

File: scout/constants/__init__.py [hunk 1]
```diff
@@ -79,8 +79,8 @@
         },
         {
             'id': 'bcftools',
-            'name': 'SAMtools',
-            }
+            'name': 'Bcftools',
+        }
```
Comment (dnil) [inline scout/constants/__init__.py]: Perhaps leave that as SAMtools for historical reasons?

**Comment by northwestwitch** [comment]
The problem if we leave bcftools with name : "SAMtools" is that SAMtools caller tag is then showed twice, like this:
image

**Comment by northwestwitch** [comment]
Ok, thanks for the review!

**Pull request merged**